Dockerfile

```diff
@@ -1,12 +1,13 @@
 # syntax=docker/dockerfile:experimental
 FROM ubuntu:22.04 as baseline
-RUN apt update && apt upgrade -y \
-  && apt install -y python3 python3-dev python3-pip curl unzip \
+
+RUN apt-get update && apt-get upgrade -y \
+  && apt-get install -y python3 python3-dev python3-pip curl unzip apt-transport-https ca-certificates gnupg \
```

Comment:
I believe we currently use `apt` instead of `apt-get` in the building of our docker images. I am not opposed to this, but again who tracking to keep this consistent across all our repos if we change here?

Reply:
@thpang, I did a quick scan and found that iac-gcp and iac-k8s are the two other projects using apt, gcp uses apt-get and iac-k8s uses both apt and apt-get. aws is using yum and azure is using apk based on their base image types.

The change to use apt-get was based on this hadolint recommendation:

./Dockerfile.trial:23 DL3027 warning: Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead

How about creating a backlog ticket to update iac-k8s so all three are in sync?